lib/resources/constants/app_constants.dart

```diff
@@ -1,5 +1,32 @@
-class AppConstants {
+/*class AppConstants {
   AppConstants._();
 
-  
+
+}*/
+import 'package:flutter/material.dart';
+
+Color kPrimaryColor =   const Color(0xFF043459);
+Color kBlackColor = const Color(0xFF181725);
+Color kWhiteColor = const Color(0xFFE5E5E5);
+Color kLightGrayColor = const Color(0xFFF2F3F2);
+Color kGrayColor = const Color(0xFF7C7C7C);
+Color kDarkGrayColor = const Color(0xFF4C4F4D);
+
+ThemeData theme = ThemeData(
+  primaryColor: kPrimaryColor,
+  backgroundColor: kWhiteColor,
+);
+
+String getAsset(String image) => 'assets/images/$image.png';
+
```

Comment:
- there are package called screen util I set up it in main.dart
- you just need to know how to use it